doc/source/api/solver/fieldinfo.rst

```diff
@@ -0,0 +1,100 @@
+.. _ref_field_info:
+
+Field info
+==========
+
+You can use field info objects to access Fluent field information.
+
+Accessing field info objects
+----------------------------
+
+In order to access field info, launch the fluent solver, and make field
+data available (for example, by reading case and data files and initializing the solver):
```

Comment:
@prmukherj `field_info` is always accessible from session instantiation, so there's no need for the caveats here.
```
>>> s2 = pf.launch_fluent(show_gui=True)
>>> fi = s2.field_info
>>> dir(fi)
['__class__', '__delattr__', '__dict__', '__dir__', '__doc__', '__eq__', '__format__', '__ge__', '__getattribute__', '__gt__', '__hash__', '__init__', '__init_subclass__', '__le__', '__lt__', '__module__', '__ne__', '__new__', '__reduce__', '__reduce_ex__', '__repr__', '__setattr__', '__sizeof__', '__str__', '__subclasshook__', '__weakref__', '_service', 'get_fields_info', 'get_range', 'get_surfaces_info', 'get_vector_fields_info']
>>> fi.get_fields_info()
{}
>>>
```

Reply:
Thank you @seanpearsonuk, I think this statement is redundant anyways, so removed it. Please let me know what you think.

Reply:
@prmukherj 

Yes, good, thanks.